src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue8788.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if !MACCATALYST
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	internal class Issue8788 : _IssuesUITest
+	{
+		public Issue8788(TestDevice device) : base(device) { }
+
+		public override string Issue => "Shell Tab is still visible after set Tab.IsVisible to false";
+
+		[Test]
+		[Category(UITestCategories.Shell)]
+		public void ShellTabItemsShouldUpdateForDynamicChangesInVisibility()
```

Comment:
The test is failing on Windows with small differences between the snapshots.
`Snapshot different than baseline: ShellTabItemsShouldUpdateForDynamicChangesInVisibility.png (1.40% difference)`
<img width="1010" height="729" alt="image" src="https://github.com/user-attachments/assets/1b3071bb-3985-48f5-8595-3dc0ca2e7b4e" />